dockerfiles/debian-stretch-all/Dockerfile

```diff
@@ -32,11 +36,6 @@ RUN curl -sL https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
          -o /usr/bin/jq \
     && chmod +x /usr/bin/jq
 
-# install latest Citus release to get header files
-RUN curl -s https://packagecloud.io/install/repositories/citusdata/community/script.deb.sh | bash \
```

Comment:
This is actually necessary for the shard rebalancer, as it relies on headers exposed by the community package (having it rely on the headers from community was easier than shoehorning an enterprise credential into our Travis build logic).

Is there a strong reason to remove this before we add Debian Stretch? That's a separate piece of work (i.e. we can do an OS audit and make sure we're not missing anything).

Reply:
There is no reason to remove them before adding Debian Stretch.

At #70 we only removed that lines for Debian Stretch and since now we add Debian Stretch it is also OK to add this back in future.